metacat-client/src/main/java/com/netflix/metacat/client/api/MetadataV1.java

```diff
@@ -116,19 +115,9 @@
     @Produces(MediaType.APPLICATION_JSON)
     void deleteDefinitionMetadata(
         @QueryParam("name")
-            QualifiedName name,
+            String name,
```

Comment:
Why start allowing any string here? If we trying to delete the catalog/db/table names even they do not exist, the 'force' parameter will allow deleting without checking if the name exists. If the records doesn't have the QualifiedName form,  we should understand why the obsolete data got into the system in the first place and build fence there.